facegame/common/static/js/fgscript.js

```diff
@@ -165,8 +167,44 @@ $(document).ready(function () {
   $('.switchimg').tipsy();
 
   new_thumbs();
+
+  $(".siteChoice").on("click", function() {
+    $(this).next().find('.glyphicon').toggleClass('active');
+    if ($(':checked').size() == 0) {
+      $('#updatebutton').prop('disabled', true);
+    } else {
+      $('#updatebutton').prop('disabled', false);
+    }
+    
+  });
 });
 
+function updatesites() {
```

Comment:
More of a generic comment than anything in this PR, but there don't seem to be any naming conventions in this repo, which is unfortunate... there's `function_name`, `functionname` and `FunctionName` at least it seems. These should really be unified but... dunno what the ROI of that would be :P